examples/custom-usage-counter/custom_usage_counter.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+import argparse
+import asyncio
+
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from decimal import Decimal
+from pathlib import Path
+
+from yapapi.strategy import LeastExpensiveLinearPayuMS, DecreaseScoreForUnconfirmedAgreement
+
+from yapapi.log import enable_default_logger
+
+from yapapi.props.base import prop, constraint
+from yapapi.props import inf, com
+
+from yapapi import Golem
+from yapapi.ctx import ActivityUsage
+from yapapi.payload import Payload
+from yapapi.services import Service, ServiceState
+
+
+@dataclass
+class CustomCounterServicePayload(Payload):
+    runtime: str = constraint(inf.INF_RUNTIME_NAME, default="test-counters")
+    min_mem_gib: float = constraint(inf.INF_MEM, operator=">=", default=0.5)
+    min_storage_gib: float = constraint(inf.INF_STORAGE, operator=">=", default=0.1)
+
+
+class CustomCounterService(Service):
+    @staticmethod
+    async def get_payload():
+        return CustomCounterServicePayload()
+
+    def __init__(self, *args, running_time_sec, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._running_time_sec = running_time_sec
+
+    async def start(self):
```

Comment:
I added explicit `start()` because without it the `DEPLOY` command was issued twice. This method can be removed when the bug is fixed.